pkg/operation/botanist/kubeapiserver.go

```diff
@@ -288,6 +296,38 @@ func (b *Botanist) DeployKubeAPIServer(ctx context.Context) error {
 	return nil
 }
 
+func (b *Botanist) computeKubeAPIServerSAConfig(externalHostname string) (kubeapiserver.ServiceAccountConfig, error) {
+	var saConfig *gardencorev1beta1.ServiceAccountConfig
+	if b.Shoot.GetInfo().Spec.Kubernetes.KubeAPIServer != nil && b.Shoot.GetInfo().Spec.Kubernetes.KubeAPIServer.ServiceAccountConfig != nil {
+		saConfig = b.Shoot.GetInfo().Spec.Kubernetes.KubeAPIServer.ServiceAccountConfig.DeepCopy()
+	}
+
+	shouldManageIssuer := v1beta1helper.HasManagedIssuer(b.Shoot.GetInfo())
+	canManageIssuer := b.Garden != nil && b.Garden.ShootServiceAccountIssuerHostname != nil && len(strings.TrimSpace(*b.Garden.ShootServiceAccountIssuerHostname)) != 0
```

Comment:
Why is this field on the `Garden`? Shouldn't it rather be on the `Shoot` since it clearly belongs to the `Shoot`? 😕

Reply:
Hmm, this is something that is configured centrally for all shoots through a secret in the Garden cluster. It was not obvious to me where this field should be set. Will move it.